rocketpy/plots/aero_surface_plots.py

```diff
@@ -71,7 +71,75 @@ def __init__(self, nosecone):
         return None
 
     def draw(self):
-        # This will de done in the future
+        """Draw the nosecone shape along with some important information,
+        including the center line and the center of pressure position.
+
+        Returns
+        -------
+        None
+        """
+        self.aero_surface.evaluate_nose_shape()
```

Comment:
If you call the draw() method N times, this will run the method N times as well, 

Maybe some try/except or even properties and cached_properties would be beneficial here, what do you think?

To be more specific, I think we should retrieve only the coords, not invoke the evaluate_nose_shape method everytime.

Reply:
This was done on purpose for a few reasons, but they are no longer valid because of the `length` change that the blufness generates.

Removed this line in https://github.com/RocketPy-Team/RocketPy/pull/413/commits/e847406d9df8eb36befc3f1d33fb9e46c25f00e4